### DISCUSSION TITLE
Overwrtie cache control on 404 page

### DISCUSSION THREAD

**Discussion by fraitag** [vercel/next.js#41389] [Help] [answered]
Hi 

Currently 404 page return ```Cache-Control: private, no-cache, no-store, max-age=0, must-revalidate```

I need to change to set max-age.

I use custom server with express and 404 is render via ```getStaticProps```

I tried to use interceptors in express but nextjs overwrite send method.

Do you have any idea?

**Comment by leerob** [comment, marked as answer]
It's now possible to customize `cache-control` as of v14.2.10. Please upgrade and give it a try!

https://github.com/vercel/next.js/pull/69802
